etl/glean_etl.py

```diff
@@ -447,11 +448,17 @@ def write_glean_metadata(output_dir, functions_dir, app_names=None):
                             )
                             metric_sample_info.get(channel)["sampled_text"] = sampled_text
 
+                            # We prefer the release channel text, but if it's
+                            # not available, we use the first available
```

Comment:
This was failing because there was no "release". So I decided to pick the first one in case, to fix it.